hedera-node/configuration/preprod/bootstrap.properties

```diff
@@ -1,5 +1,5 @@
 bootstrap.throttleDefsJson.resource=throttles.json
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```

Comment:
Does this affect previewnet?
If so we should leave this and just ensure in our perfnet tests we explicitly override this

Reply:
COnfirmed this doesn't affect previewnet